### PR TITLE
DAOS-10400 test: Supporting ucx testing in network/zero_config.py

### PR DESCRIPTION
**Pull request by phender**
Including the port number in the OFI_DOMAIN assignment for daos_racer
when testing with the UCX provider.

Skip-unit-tests: true
Test-tag: zero_config

Signed-off-by: Phillip Henderson <phillip.henderson@intel.com>

### CODE REVIEW

**Review by jgmoore-or — APPROVED** [review]

**Review by daltonbohning — DISMISSED** [review]

**Comment by phender** [comment]
The modified test is still failing with UCX so this should not land yet.

**Comment by phender** [comment]
The `network/zero_config.py` test changes are passing with verbs (no harm testing).  Verifying the change works with UCX in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8725/11/

**Comment by phender** [comment]
Updated `network/zero_config.py` test changes have been confirmed to work in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8725/16/artifact/Functional%20Hardware%20Small/network/zero_config.py/

File: src/tests/ftest/network/zero_config.py [hunk 1]
```diff
@@ -168,7 +170,14 @@ def verify_client_run(self, exp_iface, env):
         racer_env = daos_racer.get_environment(self.server_managers[0], log_file)
         racer_env["FI_LOG_LEVEL"] = "info"
         racer_env["D_LOG_MASK"] = "INFO,object=ERR,placement=ERR"
-        racer_env["OFI_DOMAIN"] = self.interfaces[exp_iface]["domain"]
+        if "ucx" in self.server_managers[0].get_config_value("provider"):
+            port_domains = [
+                ":".join([self.interfaces[name]["domain"], self.interfaces[name]["port"]])
+                for name in self.interfaces]
```
[the remark this answers is not in the document]
Comment (phender) [reply src/tests/ftest/network/zero_config.py]: Fixed

**Comment by phender** [comment]
Latest code changes passed with both verbs and ucx:

- https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8905/5/artifact/Functional%20Hardware%20Small/network/zero_config.py/results.html
- https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-8725/18/artifact/Functional%20Hardware%20Small/network/zero_config.py/results.html

**Review by jgmoore-or — APPROVED** [review]

**Pull request merged**